appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -1545,25 +1545,41 @@ definitions:
         allOf:
           - $ref: '#/definitions/Curies'
           - $ref: '#/definitions/SelfLink'
+      date:
```

Comment:
The reason for my changes is:
- In the design [http://wiki.yona.nu/display/DEV/Design+change+to+capture+app+activity] you mentioned the date a level higher, so day X [day for goal G1, day for goal G2] and week Z [week for goal G1, week for goal G2]. The date was mentioned at X and Z and not a level deeper.
- A reason is just efficiency: If you mention the date at a higher level, you do not have to mention it _n_ times at a lower level.
- Another reason is that the structure is more logical, the app builders would expect a date at the highest level instead of the same date being repeated at the children.